source/codegen/metadata/nidcpower/custom_proto.mako

```diff
@@ -0,0 +1,46 @@
+message NIComplexNumber {
+    double real = 1;
+    double imaginary = 2;
+}
+
+message NILCRMeasurement {
+  double vdc = 1;
+  double idc = 2;
+  double stimulus_frequency = 3;
+  NIComplexNumber ac_voltage = 4;
+  NIComplexNumber ac_current = 5;
+  NIComplexNumber z = 6;
+  double z_magnitude = 7;
+  double z_phase = 8;
+  NIComplexNumber y = 9;
+  double y_magnitude = 10;
+  double y_phase = 11;
+  double ls = 12;
+  double cs = 13;
+  double rs = 14;
+  double lp = 15;
+  double cp = 16;
+  double rp = 17;
+  double d = 18;
+  double q = 19;
+  uint32 measurement_mode = 20;
```

Comment:
@jonlugoatx @reckenro I used uint32 here, but in the nidcpower.h, the datatype for this in uint16. I did not find the equivalent .proto type for it [here](https://developers.google.com/protocol-buffers/docs/proto3#scalar).

Reply:
I think that makes sense. For instance, ViInt16 gets translated to sint32 for our proto codegen.

Reply:
`uint32` is the right thing to use as long as there's not an enum type for it. I looked at the DC power metadata in your other branch and didn't see a measurement mode in the enums metadata. What values are users supposed to enter here?

Reply:
I didnt find documentation for this, but I did find its usage [here](https://dev.azure.com/ni/DevCentral/_git/ni-central?version=GCbf1c90bde060727c7fef5dfbbdb1d71870c7c4db&path=%2Fsrc%2Fnidcpower%2FnidcpowerDotNETDistribution%2FMIDCPowerDotNETBinaries%2Fsource%2FNIDCPower%2FNILCRMeasurement.cs) and it seems to be an [enum](https://dev.azure.com/ni/DevCentral/_git/ni-central?path=%2Fsrc%2Fnidcpower%2FnidcpowerDotNETDistribution%2FMIDCPowerDotNETBinaries%2Fsource%2FNIDCPower%2FDCPowerInstrumentMode.cs&_a=contents&version=GBmain)

Reply:
Yes, we heard today that the LCR APIs are still under development. Since these changes were already in owners review in my PR, I incorporated the review comments. I will not be merging these changes with the main branch, I will just close this PR without submitting. I will keep my branch as is, for future reference.
FYI : @SparkingSpork @jonlugoatx @reckenro